xbmc/input/linux/LinuxInputDevices.cpp

```diff
@@ -444,6 +444,7 @@ XBMCMod CLinuxInputDevice::UpdateModifiers(XBMC_Event& devt)
     case XBMCK_RALT: modifier = XBMCKMOD_RALT; break;
     case XBMCK_LMETA: modifier = XBMCKMOD_LMETA; break;
     case XBMCK_RMETA: modifier = XBMCKMOD_RMETA; break;
+    case XBMCK_MODE: modifier = XBMCKMOD_RALT; break;
     default: break;
   }
 
```

Comment:
It would be more logical to map KEY_RIGHTALT to XBMCK_RALT.

Reply:
No - there is not right Alt key on keyboards featuring the mode key.

Reply:
Forget that, I got that wrong. You're right, it should be possible to change:
`{ KEY_RIGHTALT , XBMCK_MODE },`
to
`{ KEY_RIGHTALT , XBMCK_RALT },`
in _xbmc/input/linux/LinuxInputDevices.cpp_.